### DISCUSSION TITLE
Find user by username

### DISCUSSION THREAD

**Discussion by HairlessMom** [andygrunwald/go-jira#429] [Q&A]
Hello!
Is it possible to use the deprecated `rest/api/2/user?username=some_user`?
We have old version of Jira and it's not possible to search with accountId.

**Comment by dzaina** [comment]
same question - looks like no - I have a special jira ticket to get allocated user for that...

**Comment by andygrunwald** [comment]
Hey @HairlessMom @dzaina,
I am currently short on time for open-source contributions due to other/private obligations. If you are able to fix it, I am happy to review a pull request. Thanks for contributing to open source (also via this discussion!) and thanks for understanding.
